From ffbcaefd3febaf0c769197501e537b4af65fdd12 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 12 Apr 2018 19:19:08 +0200
Subject: [PATCH 1/1] tinker-rk3288_defconfig: Xypron specific settings

' Keep U-Boot inside 1st MB
CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x100

' Enable UEFI testing
CONFIG_CMD_BOOTEFI_HELLO=y
CONFIG_CMD_BOOTEFI_SELFTEST=y

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 configs/tinker-rk3288_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configs/tinker-rk3288_defconfig b/configs/tinker-rk3288_defconfig
index cc4350848e..c91538f1d1 100644
--- a/configs/tinker-rk3288_defconfig
+++ b/configs/tinker-rk3288_defconfig
@@ -15,9 +15,12 @@ CONFIG_CONSOLE_MUX=y
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_SPL_STACK_R=y
 CONFIG_SPL_STACK_R_MALLOC_SIMPLE_LEN=0x2000
+CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x100
 CONFIG_SPL_I2C_SUPPORT=y
 CONFIG_FASTBOOT_FLASH=y
 CONFIG_FASTBOOT_FLASH_MMC_DEV=0
+CONFIG_CMD_BOOTEFI_HELLO=y
+CONFIG_CMD_BOOTEFI_SELFTEST=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_GPT=y
 CONFIG_CMD_I2C=y
-- 
2.15.1

